posts/urls.py

```diff
@@ -17,4 +18,5 @@
     path('about/', views.about, name='posts-about'),
     path('search/', views.search, name='posts-search'),
     path('post/<int:post_pk>/comment/', CommentCreateView.as_view(), name='comment-create'),
+    path('post/<int:post_pk>/rate/', RatingCreateView.as_view(), name='rate-create'),
```

Comment:
I would change the name to 'rating-create', because rate is a verb and not a noun, so it's more logical to create a rating than "create a rate".

Reply:
Done!